src/utilities/notifications.test.ts

```diff
@@ -18,59 +12,301 @@ const baseUserInfo: UserInfo = {
 	isMe: false,
 };
 
-describe( 'getPostRestoredNotificationContent', () => {
-	it( 'should return formatted message with user name', () => {
-		const result = getPostRestoredNotificationContent( baseUserInfo );
+describe( 'notifications', () => {
+	let sendNotification: typeof import('./notifications').sendNotification;
+	let notificationTypes: typeof import('./notifications').NotificationType;
+	let createNoticeMock: any = null;
+	let getSettingMock: any = null;
 
-		assert.strictEqual( result, 'Alice restored newer content from the server.' );
-	} );
+	before( async () => {
```

Comment:
Used Selection tests as inspiration for this. It did require some experimenting to get it exactly right